src/fidesops/main.py

```diff
@@ -43,6 +45,21 @@
     app.add_exception_handler(FunctionalityNotConfigured, handler)
 
 
+@app.on_event("startup")
+async def create_webapp_dir_if_not_exists() -> None:
+    """Creates the webapp directory if it doesn't exist."""
+    if config.admin_ui.ENABLED:
+        WEBAPP_DIRECTORY = Path("src/fidesops/build/static")
+        WEBAPP_INDEX = WEBAPP_DIRECTORY / "index.html"
+        if not WEBAPP_INDEX.is_file():
+            WEBAPP_DIRECTORY.mkdir(parents=True, exist_ok=True)
+            with open(WEBAPP_DIRECTORY / "index.html", "w") as index_file:
+                index_file.write("<h1>Privacy is a Human Right!</h1>")
```

Comment:
I'm curious why is this needed? Is this just to handle a case where the docker build hasn't successfully built the admin UI and exported it, or is it more common than that?

Reply:
I followed the same pattern that the `fides` used did for this update: https://github.com/ethyca/fides/blob/main/src/fidesapi/main.py#L59-L68. I believe that it is so there is always something to display in the event that the admin UI is toggled on. 

It should probably be updated to display something for helpful to help the user know why the full admin ui isn't being displayed.

One common issue that Catherine, Sean, and I ran into while this was developed the the default config in the `docker-compose.yml` mounts a volume of the local `fidesops` files with this snippet
```yaml
    volumes:
      - type: bind
        source: ./
        target: /fidesops
        read_only: False
      - /fidesops/src/fidesops.egg-info
```

Since the admin ui is only built while the docker image is created the ui files are missing from the local `fidesops` code causing the ui files to be missing once `docker-compose up` is ran. 

This is okay because frontend development still uses `npm run dev` from `clients/admin-ui` and backend development won't typically need to use the admin ui from within the `fidesops` webserver. In the event that backend development needs the full admin within the webserver they can run the `npm run prod-export` command from within `clients/admin-ui` and that will build and move the full ui into the correct place within the backend code.